guides/CHAPTER-4.md

```diff
@@ -20,6 +20,17 @@ location: 'classpath:/flows/'
 The "location" tag is optional. If present, you can tell the system to load the flow config files from
 another folder location.
 
+## Multiple flow lists
```

Comment:
You can provide more than one flow lists to your application and it can become very handy under different situations.  For instance, to achieve better modularity in complex application, flows can be grouped to multiple categories based on development team's choice and these flows can be managed in multiple flow lists.  Another great place to use multiple flow list is to include external libraries which contain pre-defined flow lists.  The following example demonstrates that an application loads a list of flows defined in "flows.yaml" and additional flows defined in "more-flows.yaml" file of a composable library.

Reply:
developer guide updated as suggested